standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/repl/ReplScope.java

```diff
@@ -31,10 +29,10 @@
   private Pattern dbNamePattern;
 
   // Include and exclude table names/patterns exist only for REPL DUMP.
-  private List<String> includedTableNames;
-  private List<String> excludedTableNames;
-  private List<Pattern> includedTableNamePatterns;
-  private List<Pattern> excludedTableNamePatterns;
+  private String includedTableNames;
+  private String excludedTableNames;
+  private Pattern includedTableNamePatterns;
```

Comment:
Name can be includedTableNamesPattern and excludedTableNamesPattern as it is single pattern.

Reply:
done